docs/config.rst

```diff
@@ -572,6 +572,16 @@ Features
    .. _RVM: https://rvm.io/
    .. _RBENV: https://github.com/rbenv/rbenv
 
+.. attribute:: LP_ENABLE_TERRAFORM
+   :type: bool
+   :value: 1
```

Comment:
You have this default listed as enabled here, but everywhere else in the code it is disabled.